omod-common/src/test/java/org/mockingbird/test/rest/resource/BirdResource_1_9.java

```diff
@@ -60,6 +61,21 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
@dkayiwa It's your call :) ideally only a test resource should return null values. Other resource will either implement or could just inherit from its parent